storybook/scripts/prepare-manager-head-file.ts

```diff
@@ -0,0 +1,20 @@
+const envVars: Record<string, string | undefined> = {
+  /** ENV's needed for CookieConsent script integration */
+  COOKIELAW_ORG: process.env.COOKIELAW_ORG,
+  /** ENV's needed for FullStory integration */
+  FS_HOST: process.env.FS_HOST,
+  FS_SCRIPT: process.env.FS_SCRIPT,
+  FS_ORG: process.env.FS_ORG,
+  /** ENV's needed for UserGuiding integration */
+  USERGUIDING_ORG: process.env.USERGUIDING_ORG,
```

Comment:
are these env vars set in vercel?  When I build storybook for production and serve it, I get a 404 for this URL:
`https://cdn.cookielaw.org/consent//OtAutoBlock.js`, which I assume means that the `COOKIELAW_ORG` env var isnt set and this script is failing:

```<script type="text/javascript" src="https://cdn.cookielaw.org/consent/{{COOKIELAW_ORG}}/OtAutoBlock.js"></script>```